doc/source/configuring.rst

```diff
@@ -218,3 +232,35 @@ regular expression ``notify-.*@disqus.net``.
          regex: "notify-.*@disqus.net"
      action:
        name: delete
+
+Complete example configuration file
+===================================
+
+Here's an example of a configuration file with all the possible parts.
+
+.. code-block:: yaml
+
+    server:
+      hostname: imap.gmail.com
+      username: user@example.com
+      password: xxxxxxxxxxxxxx
+
+      trash-mailbox: "[Gmail]/Trash"
```

Comment:
I think trash-mailbox is supposed to be a top-level key, isn't it? Not indented under the server key?

Reply:
I would think it'd be a server configuration, if nothing else to be forward-compatible to a possible multi-server mode of running this. Different servers have different conventions for trash folders.

Reply:
Yeah, I think I messed that up in the initial implementation/design because I'm only using the tool with one mail account. So it *should* be server-specific but right now it isn't.